File: conexao/Conexao.php
<?php

class Conexao{

	function criarConexao() {
		try { 
		    $host = "http://onebus.cx8kmic8cryz.us-east-2.rds.amazonaws.com/";   //18.216.171.234
			$username = "admin";		
			$password = "onebus123";
			$db_name = "onebus";

			$con = mysqli_connect ( "$host", "$username", "$password" ) or die ( "cannot connect; "+mysqli_error($con) );
			mysqli_select_db ( $con, "$db_name" ) or die ( "cannot select DB; "+mysqli_error($con) );
            mysqli_set_charset ( $con , "utf8" );
            
			return $con;

		} catch ( Exception $e ) {
			print "Error!: " . $e->getMessage() . "<br/>";
			die();
			return null;
		}
	}

}

?>

File: conexao/Conexao.py
import mysql.connector
from mysql.connector import Error

class Conexao:

    def criar_conexao(self):
        try:
            host = "onebus.cx8kmic8cryz.us-east-2.rds.amazonaws.com"
            username = "admin"
            password = "onebus123"
            db_name = "onebus"

            con = mysql.connector.connect(
                host=host,
                user=username,
                password=password,
                database=db_name
            )

            if con.is_connected():
                print("Conexão bem-sucedida!")
                con.set_charset_collation('utf8')
                return con

        except Error as e:
            print(f"Erro: {e}")
            return None
